### DISCUSSION TITLE
blog/2024/01/do-you-really-need-a-/home-partition/

### DISCUSSION THREAD

**Discussion by giscus** [userliluzibert/userliluzibert.github.io#7] [Giscus]
# blog/2024/01/do-you-really-need-a-/home-partition/

So I use arch (btw), and I recently had to do a few reinstalls (wrote about in my last blog). This last fuck up was because I was trying to install some big files, but I partitioned it wrong. I trusted the archinstall script to basically do everything for me, and it separated the home partition. When it did that, there wasn’t enough space in usr/share for it to download anymore packages.

https://liluzibird.github.io/blog/2024/01/do-you-really-need-a-/home-partition/

**Comment by Ast3risk-ops** [comment]
As much as I like this theme, I've left it for [Blowfish](https://blowfish.page).

https://asterisk.lol

It's been fun.

**Reply by liluzibird** [reply]
Lol. I'm skimming the people using that theme and it looks pretty cool. I still like this HBS theme though because it's pretty simple, and I do enjoy the timeline feature.

I've had issues with getting my RSS feed for this site display the entire blog instead of partially. Do you have any clue how to fix that xml template? I don't even know where it's located.

**Reply by Ast3risk-ops** [reply]
Blowfish has something similar but I will agree HBS is easier to set up, I spent several days on this masterpiece. Worth every minute.

Not sure about how the RSS feed works due to poor documentation. It always worked for me (at least for posts, I don't use the docs feature). the compiled file is located at `/index.xml`
